spec/factories/upload_files.rb

```diff
@@ -0,0 +1,6 @@
+FactoryGirl.define do
+  factory :upload_file do
+    path "MyString"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.